src/server/world.zig

```diff
@@ -120,46 +120,44 @@ pub fn tryCreateWorld(worldName: []const u8, worldSettings: WorldSettings) !void
 }
 
 pub const EntityChunk = struct {
+	const State = packed struct(u8) {
+		pub const NotLoaded: State = .{.loaded = false, .touched = true};
+		pub const Loaded: State = .{.loaded = true, .touched = true};
+		pub const NotTouched: State = .{.loaded = true, .touched = false};
+		pub const Touched: State = .{.loaded = true, .touched = true};
```

Comment:
I think these aliases don't particularly help me understand the flow here. To me it looks like there are illegal state changes happening in various places.